app/controllers/press-releases/press-release.js

```diff
@@ -4,9 +4,58 @@ import { action } from '@ember/object';
 import { task } from 'ember-concurrency-decorators';
 
 export default class PressReleasesPressReleaseController extends Controller {
-  @tracked showPublishDropdown = false;
   @tracked showPublicationModal = false;
   @tracked showPublicationPlanningModal = false;
+  @tracked showConfirmationModal = false;
+
+
+  get snapshot() {
+    return this.model;
+  }
+
+  get pressRelease() {
+    return this.snapshot.pressRelease;
+  }
+
+  @task
+  *savePressRelease() {
+    yield this.snapshot.save();
+  }
+
+  @action
+  saveChanges() {
+    this.savePressRelease.perform();
+  }
+
+  @action
+  async saveChangesAndNavigateBack() {
+    await this.savePressRelease.perform();
+    if (this.savePressRelease.last.isSuccessful) {
```

Comment:
Deze check heeft niet zo veel zin. Indien `this.savePressRelease.perform()` op de vorige lijn een fout zou geven, gooit die task een exception en ga je deze lijn dus nooit bereiken. 
Het weergeven van een error toast voor de gebruiker volgt later nog, dus dat moet je hier niet toevoegen.